pages/glossary/automatic-error-detection.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Automatic detection of input errors
+key: automatic-error-detection
+---
+
+Automatic detection of [input errors](https://www.w3.org/TR/WCAG21/#dfn-input-error) is the result of:
```

Comment:
This entire definition is quite vague in my opinion.

We either need links to examples of what a `specific format` is or to some best practises on how a form field should be filled.

Can we make do without this defn?

Reply:
I've tried to improve this, including exemples